README.md

```diff
@@ -113,4 +113,20 @@ Additionally, a device simulator should be present in the system to serve as a r
 
 >Below is a sample scheme of what the simulator should provide (image is illustrative):
 
-![Simulator UI](doc_resources/sample_simulator_overview.png)
\ No newline at end of file
+![Simulator UI](doc_resources/sample_simulator_overview.png)
+
+
+
+## Setup running environment
+
+### For angular:
+
+Dependencies: NODEjs (npm), Angular2
+1. Install Angular2 from node: npm install -g @angular/cli
```

Comment:
You can put all commands in backtick quotes (``) so that they are rendered as code e.g. `npm install`,